web-app/.env.prod.dist

```diff
@@ -2,4 +2,5 @@ TYPEORM_CONNECTION=postgres
 TYPEORM_HOST=postgres
 TYPEORM_USERNAME=postgres
 TYPEORM_PASSWORD=postgres
-TYPEORM_DATABASE=test
\ No newline at end of file
+TYPEORM_DATABASE=test
+JWT_SECRET=aaabbbccc
```

Comment:
Are you sure you want to push confidential information on the depot?

Reply:
The philosophy of this file is to give an example value for each param, as we could/should have them in production. But the real values are created from a github secret, don't worry 😉 .

As I side note, on second thought I also agree that having this file is arguable. Maybe the `.env.dev.dist` file is enough to serve as a template...

Reply:
Yes, we should probably delete the prod one !